src/routes.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
```

Comment:
Yeah, I think this would be related to comment on middleware. If fails middleware check, just redirect to this page? Not exactly sure how that would actually work though..